### PR TITLE
ci: add Travis CI config with some default settings

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+cache:
+  directories:
+    - "${HOME}/.composer/cache/files"
+
+install:
+  - composer install
+
+script:
+  - composer test
```
Comment (shyim) [inline .travis.yml:27]: This repository currently don't have tests, so there would nothing happends :D

**Review by shyim — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (DanielRuf) [reply .travis.yml]: Same in another plugin of FrindsOfShopware ;-)

It exists with 0 if there is no test so this is fine. If there will be tests in the future, the are automatically picked up by PHPUnit

Comment (DanielRuf) [reply .travis.yml:27]: Same in another plugin of FriendsOfShopware ;-)

It exists with 0 if there is no test so this is fine. If there will be tests in the future, the are automatically picked up by PHPUnit

Comment (DanielRuf) [reply .travis.yml:27]: Oh and Travis CI has to be enabled for PRs and commits.

Comment (shyim) [reply .travis.yml:27]: It makes no sense to me. If no tests are available, the ci will everytime fail

Comment (DanielRuf) [reply .travis.yml:27]: Are you sure?
https://github.com/FriendsOfShopware/FroshViewSnapshots
https://travis-ci.org/FriendsOfShopware/FroshViewSnapshots/jobs/368786388#L1266-L1269

https://stackoverflow.com/questions/40421726/phpunit-should-fail-if-no-test-is-executed

**Comment by shyim** [comment]
I added one for the issue #5 . But thanks for the pull Request

**Pull request closed without merging**